tests/tests_unit/qhub-config-yaml-files-for-upgrade/qhub-config-aws-310-customauth.yaml

```diff
@@ -73,6 +73,7 @@ amazon_web_services:
       gpu: false
       single_subnet: false
       permissions_boundary:
+      capacity_type: "SPOT"
```

Comment:
Those files (prefix `qhub`) should not be changed, they are used for legacy migration testing